forge/db/models/Project.js

```diff
@@ -577,6 +577,16 @@ module.exports = {
                             }
                         ]
                     })
+                },
+                countByState: async (states, teamId) => {
```

Comment:
You are passing the hashId from the api through - this will need converting to the database id